test/blackbox-tests/test-cases/directory-targets/cache-permissions.t

```diff
@@ -0,0 +1,56 @@
+We expect a clean restore when some directory targets have modes like 755.
+
+See #11533.
+
+This test relies on a particular umask.
+
+  $ umask 002
+
+  $ export DUNE_CACHE_ROOT=$PWD/.cache
+  $ export DUNE_CACHE=enabled
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.0)
```

Comment:
```suggestion
  > (lang dune 3.17)
```
Why not `3.17`?

Reply:
This test doesn't use anything from the 3.17 rules. Using a lower version makes backporting and bisecting easier.